test/integration/util/util.go

```diff
@@ -120,7 +122,7 @@ type OcRunner struct {
 }
 
 func NewOcRunner() *OcRunner {
-	p, _ := exec.LookPath("oc")
+	p := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "oc", "v1.4.1", "oc")
```

Comment:
@praveenkumar , 
I think we can cache the common dirs like `oc` dir, as 

```
OcDir := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "oc")
ISODir := path.Join(os.Getenv(constants.MiniShiftHomeEnv), "cache", "iso")
```

etc.

Doesn't feel good to add testDir to $PATH. This also might not work in Windows but unsure.